### PR TITLE
Add EnderCrystal

### PR DESCRIPTION
**Pull request by PikyCZ**

### CODE REVIEW

**Comment by kvetinac97** [comment]
Good, but there is "ShowBottom" metadata missing (which makes the "Half-bedrock" under the crystal)

**Comment by PikyCZ** [comment]
can you add this?

**Comment by kvetinac97** [comment]
It's a simple metadata, however, I don't know exact type (Byte/Boolean?) and number of it.

**Comment by PikyCZ** [comment]
number: 7	
type: Boolean	
Show bottom	is true

**Comment by kvetinac97** [comment]
Nope, it does not work. Just made it 7 - true, but nothing changed. Still no "bedrock-half"

**Comment by PikyCZ** [comment]
Try 95  .. its invisible bedrock..

**Comment by Tee7even** [comment]
It's not ready to merge yet, right?

**Comment by PikyCZ** [comment]
EnderCrystals is ok :D

 but "ShowBottom" metadata missing

**Comment by Tee7even** [comment]
`EntityLiving`?

File: src/main/java/cn/nukkit/entity/EntityEnderCrystal.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+import cn.nukkit.Player;
+import cn.nukkit.event.entity.ExplosionPrimeEvent;
+import cn.nukkit.level.Explosion;
+import cn.nukkit.level.format.FullChunk;
+import cn.nukkit.math.Vector3;
+import cn.nukkit.nbt.tag.CompoundTag;
+import cn.nukkit.network.protocol.AddEntityPacket;
+
+public class EntityEnderCrystal extends EntityLiving implements EntityExplosive {
```
Comment (Tee7even) [inline src/main/java/cn/nukkit/entity/EntityEnderCrystal.java]: `EntityLiving`? Are you sure?

Comment (kvetinac97) [inline src/main/java/cn/nukkit/entity/EntityEnderCrystal.java]: Tee7even could you find the metadata for ShowBottom?

Comment (Snake1999) [inline src/main/java/cn/nukkit/entity/EntityEnderCrystal.java]: @kvetinac97 Be serious about everything you're talking. If you are always asking without contributing like this, we will block you outside from this project.

Comment (PikyCZ) [inline src/main/java/cn/nukkit/entity/EntityEnderCrystal.java]: hmm

**Comment by caspervanneck** [comment]
@PikyCZ  they can only on obsidian and bedrock

**Comment by caspervanneck** [comment]
on vanila

File: src/main/java/cn/nukkit/entity/EntityEnderCrystal.java (hunk 1 above)
Comment (Creeperface01) [inline src/main/java/cn/nukkit/entity/EntityEnderCrystal.java]: xD

**Comment by caspervanneck** [comment]
@PikyCZ  !!!!!!!!!!!!!!! they can only set on obsidian and bedrock

**Comment by Creeperface01** [comment]
@caspervanneck what

**Comment by PikyCZ** [comment]
@caspervanneck  i know ..

**Pull request closed without merging**